testdata/project-v3-v1beta1/Makefile

```diff
@@ -2,7 +2,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.24.1
+ENVTEST_K8S_VERSION = 1.21
```

Comment:
is this going backwards? 1.24.1 -> 1.21?

Reply:
Yes. A project that is scaffold with v1beta1 cannot work with k8s >= 1.22 so that cannot work with 1.24.1.
This sample is done using the crdversions v1beta1 and when this option is used we downgrade the scaffold to warn the users.

Reply:
Note that the sample testdata/project-v3-v1beta1 is built using CRD v1beta1 and the optional flag crd-version=v1beta1

Reply:
gotcha!